### PR TITLE
Add integration test for accelerateTx

### PR DESCRIPTION
**Pull request by OttoAllmendinger**
Currently uses latest.bitgo.com

Fails with `Error: route not found` due to outdated IMS version that
does not support the `/select-cpfp` route.

Issue: BG-9646

### CODE REVIEW

File: test/v2/integration/accelerateTx.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  }));
+
+  it('can accelerate unconfirmed tx', co(function *() {
+    console.log('acclerateTx..');
+    const result = yield wallet.accelerateTransaction({
+      cpfpTxIds: [unconfirmedTx],
+      cpfpFeeRate,
+      maxFee: 1000000
+    });
+    console.log('result', result);
```
Comment (paddyquinn) [inline test/v2/integration/accelerateTx.js]: Is it possible to deterministically calculate the fee rate here such that we can test that this is working properly?

File: test/v2/integration/accelerateTx.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      feeRate: baseFeeRate,
+      walletPassphrase
+    });
+    return transfer.txid;
+  }.bind(this));
+
+  let unconfirmedTx;
+
+  before(co(function *() {
+    this.timeout(0);
```
[the remark this answers is not in the document]
Comment (OttoAllmendinger) [reply test/v2/integration/accelerateTx.js]: this takes a lot of time so I disabled timeouts for now

File: test/v2/integration/accelerateTx.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+require('should');
+const co = require('bluebird').coroutine;
+
+const BitGoJS = require('../../../src');
+
+describe('AccelerateTx', function() {
+  // latest.bitgo.com / otto-latest@bitgo.com
+  const accessToken = 'v2x0f5a14ab6aa4c3d4a420444ed6494157b7259de634ad3836ba748335812c6252';
```
[the remark this answers is not in the document]
Comment (paddyquinn) [reply test/v2/integration/accelerateTx.js]: Great. Resolved.

File: test/v2/integration/accelerateTx.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (paddyquinn) [reply test/v2/integration/accelerateTx.js]: Gotcha. Resolved.

File: test/v2/integration/accelerateTx.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+    console.log('creating unconfirmed tx...');
+    unconfirmedTx = yield getUnconfirmedTx(wallet);
+    console.log('created tx', unconfirmedTx);
+    (unconfirmedTx === undefined).should.eql(false);
+  }));
+
+  it('can accelerate an unconfirmed tx', co(function *() {
+    console.log('acclerateTx...');
+    // FIXME: the tx is not found in the db when we go to access it
```
Comment (OttoAllmendinger) [inline test/v2/integration/accelerateTx.js]: I found waiting 1000ms on my system changes the error to `cannot build tx without available unspents`

I will add a delay with 2000ms to be on the safe side

**Comment by tylerlevine** [comment]
@paddyquinn Is this PR still active?

**Comment by paddyquinn** [comment]
These integration tests should also be merged in and there is a ticket for them but in its current state this branch needs to be cleaned up before that can be done so I'll close this for now.

**Pull request closed without merging**